### ISSUE TITLE
[BUG] Missing focus outline for UiRating component

### ISSUE THREAD

**Issue by mateuszkulpa** [infermedica/component-library#180]
There is no outline when UiRating item is focused.

Reproduction:
1. Open story: https://main--639728541e2039caadfe5404.chromatic.com/?path=/story/molecules-rating--common
2. Try to navigate through rating items

Current result:
Outline is not visible when the rating item is focused

Expected result:
Outline is visible when the rating item is focused
